File: Algorithms/Implementation/Breaking the Records.php
<?php

/*
 * Complete the 'breakingRecords' function below.
 *
 * The function is expected to return an INTEGER_ARRAY.
 * The function accepts INTEGER_ARRAY scores as parameter.
 */

function breakingRecords($scores) {
    # Write your code here
    $min = $scores[0];
    $max = $scores[0];
    $min_count = 0;
    $max_count = 0;
    for($i = 1; $i < count($scores); $i++) {
        if($scores[$i] > $max) {
            $max = $scores[$i];
            $max_count++;
        }
        if($scores[$i] < $min) {
            $min = $scores[$i];
            $min_count++;
        }
    }
    return [$max_count,$min_count];
}

$fptr = fopen(getenv("OUTPUT_PATH"), "w");

$n = intval(trim(fgets(STDIN)));
$scores_temp = rtrim(fgets(STDIN));
$scores = array_map('intval', preg_split('/ /', $scores_temp, -1, PREG_SPLIT_NO_EMPTY));

$result = breakingRecords($scores);

fwrite($fptr, implode(" ", $result) . "\n");
fclose($fptr);

File: Algorithms/Implementation/Breaking the Records.py
import os

#
# Complete the 'breakingRecords' function below.
#
# The function is expected to return an INTEGER_ARRAY.
# The function accepts INTEGER_ARRAY scores as parameter.
#

def breakingRecords(scores):
    # Write your code here
    highest, lowest = scores[0], scores[0]
    highest_count, lowest_count = 0, 0
    for score in scores:
        if score > highest:
            highest = score
            highest_count += 1
        elif score < lowest:
            lowest = score
            lowest_count += 1
    return highest_count, lowest_count


if __name__ == '__main__':

    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    n = int(input().strip())

    scores = list(map(int, input().rstrip().split()))

    result = breakingRecords(scores)

    fptr.write(' '.join(map(str, result)))
    fptr.write('\n')

    fptr.close()
